package EquipSmeltConfig;

message CharacterLevel
{
	required uint32 ch			= 1;   //品质
	required uint32 maxlevel	= 2;   //最大洗练等级
}

message SuitLevel
{
	required uint32 type		= 1;  //套装类型
	required uint32 maxlevel	= 2;  //最大洗练等级
}

message LevelProperty
{
	required uint32 level		= 1;
	repeated uint32 property	= 2;
}

message EquipSmelt
{
	repeated CharacterLevel chlevel		= 1;   //品质与洗练等级上限关系
	repeated uint32 		normal		= 2;   //普通洗练概率
	repeated uint32 		super		= 3;   //至尊洗练概率
	repeated LevelProperty levelpro		= 4;   //属性等级与值关系
	repeated SuitLevel      suitlevel	= 5;   //套装类型与洗练等级上限关系
}